course05/homework01/martin/string.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <string>
+#include <ostream>
+#include <iostream>
+
+class String
+{
+public:
+	String();
+	String(std::string);
+
+
+        bool Empty() const;
+        std::size_t Size() const;
+        int SetString(std::string str);
+        std::string GetString() const;
+
+private:
+        std::string mStr;
+
+};
+
+String::String()
+{
+    std::cout << "String has been initialized" << std::endl;
+}
+
+bool String::Empty() const
+{
+    return mStr.empty();
+}
+
+int String::SetString(std::string str)
+{
+    mStr = str;
+    std::cout << mStr << std::endl;
+    return 0;
```

Comment:
What's the point of returning 0 ? This can be very surprising / cause bugs, I was expecting the length of new string being returned for example